frontend/src/metabase/models/containers/FormModelPicker/FormModelPicker.tsx

```diff
@@ -39,6 +38,11 @@ function FormModelPicker({
   const [{ value }, { error, touched }, { setValue }] = useField(name);
   const formFieldRef = useRef<HTMLDivElement>(null);
   const [width, setWidth] = useState(MIN_POPOVER_WIDTH);
+  const isActionFromModel = typeof value === "number";
```

Comment:
I think this is a bit misleading, Actions are always tied to a model. I believe this check is for when no model is selected yet, so we would display a placeholder instead. e.g. when creating a new model via `+ New` button in the top right of the screen > `Action`
![image](https://github.com/metabase/metabase/assets/1937582/1912e3f2-309b-4d79-b488-264e7ff4eab8)

I think the name could be something along the line
```suggestion
  const isModelSelected = typeof value === "number";
```

Reply:
The action can be created from the top "+ New" button, in this case you don't have a model preselected 
<img width="1698" alt="image" src="https://github.com/metabase/metabase/assets/125459446/c8a71a1d-88c7-46cf-a69e-b415c7cddf47">

I'm open for a better name, but I also want to highlight if there is a model already associated with the action. `isModelSelected` is fine, maybe `shouldPreselectModel`?

Reply:
what about `shouldInitiallyPreselectModel`? 😄 
`isModelSelected` - I can say the same - user didn't select the model, so why should we call it like this.

at the same time having in the code `typeof value === 'number'` is not self explained anyway

Reply:
hasValue doesn't show any business value here, it just tries to rephrase `typeof value === 'number'` IMO

Reply:
Alright, @WiNloSt I appreciate your patience and push for improvement, the var is changed